packages/dvilela/skills/memeooorr_abci/payloads.py

```diff
@@ -115,6 +117,27 @@ class CallCheckpointPayload(BaseTxPayload):
     tx_hash: Optional[str]
 
 
+@dataclass(frozen=True)
+class PostMechRequestPayload(BaseTxPayload):
+    """Represent a transaction payload for the PostMechRequestRound."""
+
+    mech_for_twitter: bool
+
+
+@dataclass(frozen=True)
+class FailedMechResponsePayload(BaseTxPayload):
+    """Represent a transaction payload for the PostMechResponseRound."""
+
+    mech_for_twitter: bool
+
+
+@dataclass(frozen=True)
+class FailedMechRequestPayload(BaseTxPayload):
+    """Represent a transaction payload for the PostMechResponseRound."""
+
+    mech_for_twitter: bool
+
+
```

Comment:
No need to have 3 payloads with the same content; could only use one of them for all the rounds.